PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,119 @@
+
+There are exceptions with all guidelines. As long as your decisions are justified, then you are good! Contact the reviewers or the leads about any exceptions.
+
+**Please *read* every word on *every* bullet point before checking off the corresponding box.**
+
+## Minimum Prerequisites
+- [ ] The board passes ERC and DRC.
+    - Note some ERC warnings are acceptable (**_CAUTION_** please be careful)
+    - Note some DRC warnings are acceptable (less so than ERC: **_CAUTION_** please be careful).
+- [ ] All traces are routed.
+- [ ] Units are in metric.
+- [ ] Gerbers are in a zip file in the main directory (after physical design is done)
+
+**Please *read* every word on *every* bullet point before checking off the corresponding box.**
+
+## Schematic Level Requirements 
+- [ ] Is proper noise resistance given to all peripheral devices (bypass caps and coils/ferrites)?  
+- [ ] Is proper ESD protection given to all MCU input pins (zener diodes)?  
+- [ ] Is proper power protection given to peripheral devices (zener diodes)?  
+- [ ] Are peripheral units used properly (reading datasheet)?  
+- [ ] Are testing points added at useful places?  
+- [ ] Is there proper short to GND protection at MCU outputs (inline resistors)?  
+- [ ] Do ADC inputs have caps?  
+- [ ] Are ADC inputs biased so there is room above expected value to determine if value is being overflowed?  
+- [ ] Are LED's located at useful places (comm, power, debugging, extra GPIO)?  
+- [ ] Are parts chosen easy to collect?  
+- [ ] Are parts chosen easy to solder?  
+- [ ] Is there reverse polarity protection on inputs?  
+- [ ] Do parts in the schematic match the actual part datasheet?
+
+**Please *read* every word on *every* bullet point before checking off the corresponding box.**
+
+## BOM Requirements
+- [ ] **Bill Of Materials**: is there an Excel Sheet BOM and a Mouser Cart link?
+- [ ] Are *all* the correct parts in the BOM?
+- [ ] Is each part in the Mouser Cart in stock (or do we have the part in stock)?
+- [ ] Does each part in the Mouser Cart have a margin for part quantity?
+- [ ] Does every component in the schematic have a footprint, mouser link/datasheet, and reference number associated with it?
+
+**Please *read* every word on *every* bullet point before checking off the corresponding box.**
+
+## Layout Level Requirements 
+### 2D Spacing
+- [ ] The components are spaced out at an optimal distance.
+    - All components can be easily hand-soldered.
+    - IPC-SM-782A Standard requires a minimum distance of 1.0mm from edge to edge.
+- [ ] Components that are in parallel with each other are spaced out at an equal distance when possible.
+- [ ] The components are aligned with each other when possible.
+- [ ] Components are grouped based off of functionality.
```

Comment:
Bless KiCAD pack and move, should make sure to teach this in the LHRs kicad tutorials

Reply:
Is that just selecting multiple and moving them?

Reply:
very neat feature
https://www.kicad.org/video_clips/blog/2022/pack-footprints.mp4